src/app/helpers/README.md

```diff
@@ -0,0 +1 @@
+Currentlt we do not differentiate between helpers and utils, put them all here. The only exception is models which have their own directory.
```

Comment:
A brief explanation of what falls under helpers would be great. Especially distinguishing it from things that should be in the `lib` directory.

Reply:
Also there's a typo - `Currently`